wgpu-core/src/validation.rs

```diff
@@ -263,6 +263,8 @@ pub enum StageError {
         #[source]
         error: InputError,
     },
+    #[error("location[{location}] is provided by the previous stage output but is not sonsumed as input by this stage.")]
```

Comment:
Just noticed a typo `sonsumed`

Reply:
Name a more iconic duo than cwfitzgerald and typos.

Thanks!